### PR TITLE
MAKES PLAYER GORILLAS OOGA OOGA WHEN ATTACKA

### PR DESCRIPTION
**Pull request by ShizCalev**
OOGA OOGA OOGA

### CODE REVIEW

**Comment by Ratvar** [comment]
GORILLA THEMED HOLOPARASYTES WHEN

**Comment by FantasticFwoosh** [comment]
- He attac
- He protec
- But most of all he OOGA

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 1]
```diff
@@ -49,6 +50,8 @@
 			return parts
 
 /mob/living/simple_animal/hostile/gorilla/AttackingTarget()
+	if(src.client)
+		oogaooga()
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: Remove src

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Grazyn** [comment]
isn't it ooga BOOGA?

**Comment by Shadowlight213** [comment]
This is already spammed so much though

**Comment by ShizCalev** [comment]
@Shadowlight213 It isn't spammed by player ones, though.

**Comment by Tacolizard** [comment]
>var/oogas

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 2]
```diff
@@ -34,6 +34,7 @@
 	maxbodytemp = 350
 	unique_name = TRUE
 	var/list/gorilla_overlays[GORILLA_TOTAL_LAYERS]
+	var/oogas = 0
```
Comment (Tacolizard) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: remove the = 0

**Review by Tacolizard — APPROVED** [review]

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm (hunk 2 above)
Comment (RemieRichards) [reply code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: No, leave the ```= 0```

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**